examples/arNFT_gltf_brave_robot_example.html

```diff
@@ -24,14 +24,19 @@
 				document.addEventListener('getThreejsRenderer', (ev) => {			
 				    let renderer = ev.detail.renderer.renderer
 				    renderer.outputEncoding = THREE.sRGBEncoding;
-				    renderer.physicallyCorrectLights = true;
+					renderer.physicallyCorrectLights = true;
+					console.log('setting texture outputEncoding and physicallyCorrectLights');
 				})
-				document.addEventListener('getInitThreejsRenderer', (ev) => {
+				document.addEventListener('onInitThreejsRendering', (ev) => {
```

Comment:
**onInitThreejsRendering** -> will give you access to the renderer, scene and camera, you can make all your mod you want.
**getThreejsRenderer** -> happens after `onInitThreejsRendering`, see in the console, maybe better to re-name as `onAfterThreejsRendering`

Reply:
i prefered to rename it as `onAfterInitThreejsRendering`, i think it is more appropriate.